recommendation_engine/pmf/train_pmf.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Train the probabilistic matrix factorization model."""
+
```

Comment:
All the training files are already a part of the other pull request. Please remove them from here.

Reply:
ok